File: fizzbuzz/fizzbuzz.php
<?php

function fizzbuzz($fizz, $buzz, $count)
{
    $result = "";

    for ($i = 1; $i <= $count; $i++)
	{
		if ($i % $fizz == 0 && $i % $buzz == 0)
		{
            $result .= "FB ";
		}
		elseif ($i % $fizz == 0)
		{
            $result .= "F ";
		}
		elseif ($i % $buzz == 0)
		{
            $result .= "B ";
		}
		else
		{
            $result .= "$i ";
		}

	}

	$result = trim($result);

	return $result;
}

$fh = fopen($argv[1], "r");

while (!feof($fh))
{
  $test = trim(fgets($fh));

  if ($test != "")
  {
    $file_data = explode(" ", $test);

    $fizz = $file_data[0];
    $buzz = $file_data[1];
    $count = $file_data[2];

	$result = fizzbuzz($fizz, $buzz, $count);

	echo $result . "\n";
  }
}
?>

File: fizzbuzz/fizzbuzz.c
#include <stdio.h>
#include <string.h>

int main(int argc, const char * argv[]) {
    FILE *file = fopen(argv[1], "r");
    char line[1024];
	char *pch;

    while (fgets(line, 1024, file)) {
	    pch = strtok (line, " ");
        int fizz = atoi(pch);

		pch = strtok (NULL, " ");
		int buzz = atoi(pch);

		pch = strtok (NULL, " ");
		int counter = atoi(pch);

		int i;

		for (i = 1; i <= counter; i++)
		{
		    if (i % fizz == 0 && i % buzz == 0) { printf("FB "); }
			else if (i % fizz == 0) { printf("F "); }
			else if (i % buzz == 0) { printf("B "); }
			else { printf("%i ", i); }
		}

        printf("\n");
    }

    return 0;
}
